src/ja/user/config/query.py

```diff
@@ -12,70 +13,297 @@ class QueryCommandConfig(UserConfig):
     matches any of the provided values.
     """
 
+    def __init__(self, uid: List[str] = None, label: List[str] = None, owner: List[str] = None,
+                 priority: List[JobPriority] = None, status: List[JobStatus] = None, is_preemtible: bool = None,
+                 special_resources: List[List[str]] = None, cpu_threads: Tuple[int, int] = (1, 1),
+                 memory: Tuple[int, int] = (1, 1), before: datetime = None, after: datetime = None,
```

Comment:
(1, 1) are bad default values for cpu_threads and memory because they are included in the range of valid values.
Please replace them with something like None, (None, None), or (-1, -1).